HHS/Head-Start-TTADP
# [TTAHUB-1829] Session report expander

## Description of change

Add expander to view session reports via the event on the training reports landing.


## How to test

View the training report landing. If a event has atleast one session report we should show and expander that allows the user to view the underlying sessions (similar to objectives on goals).

Clicking the edit session link with the correct permissions should redirect to the session edit page. Right now we only show the edit button if they have read/write permissions.

https://www.figma.com/file/RZqI76V9igIpWUkHB2KdUm/PD-Event-Form?type=design&node-id=262-39406&mode=design&t=W636jewI1Wuc1OfB-0

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-1829


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
